keyboards/crkbd/keymaps/oo/README.md

```diff
@@ -0,0 +1,14 @@
+
+# Build instructions
```

Comment:
```suggestion
# Build instructions
```
And could you please rename this file to lowercase?

Reply:
Done and done, @fauxpark!